source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -171,7 +172,7 @@ Filter::StreamOpenState Filter::openStream() {
     stats_.streams_started_.inc();
     // For custom access logging purposes. Applicable only for Envoy gRPC as Google gRPC does not
     // have a proper implementation of streamInfo.
-    if (grpc_service_.has_envoy_grpc() && logging_info_ != nullptr) {
+    if (grpc_service_.has_envoy_grpc()) {
```

Comment:
FilterStateImpl can return nullptr. https://github.com/envoyproxy/envoy/blob/628a5b997ef36fe2b76624797a693970461a637a/source/common/stream_info/filter_state_impl.cc#L83

In general it s a good defensive pattern to check for nullptr when dereferencing the pointer. It has 0 cost and can save from outage.